src/chapter_2_multi-serviced_approach.md

```diff
@@ -0,0 +1,46 @@
+# Services
+
+Twilight is built with a service-minded approach. This means that it caters to
+both monolithic and multi-serviced applications equally. If you have a very
+large bot and have a multi-serviced application and feel like Rust is a good
+language to use for some of your services, then Twilight is a great choice. If
+you have a small bot and just want to get it going in a monolithic application,
+then it's also a good choice. It's easy to split off parts of your application
+into other services as your application grows.
+
+## Gateway clusters
+
+One of the popular design choices when creating a multi-serviced application is
+to have a service that simply connects shards to the gateway and sends the
+events to a broker to be processed. As bots grow into hundreds or thousands of
+shards, multiple instances of the application can be created and clusters -
```

Comment:
hundreds of thousands of `shards` or `guilds`?

Reply:
Shards is correct here